unlock-js/src/__tests__/integration/web3ServiceIntegration.test.js

```diff
@@ -1,11 +1,8 @@
-import nock from 'nock'
```

Comment:
This was not used (because of course this is integration tests so we actually do not want to mock things!)